src/dstore/dstore_base.c

```diff
@@ -799,3 +757,125 @@ int dstore_pread(struct dstore_obj *obj, off_t offset, size_t count,
 
 	return rc;
 }
+
+int dstore_dealloc_op(struct dstore_obj *obj, struct dstore_io_vec *vec,
```

Comment:
Make this API internal... `static int ...` same for `dstore_deallocate` too.

Reply:
Done.